src/components/SelectButtonGroup/SelectButton/SelectButton.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+interface Props {
+    /** the label to be displayed on this button */
+    children: string;
+    /** the value associated with this button */
+    value: string;
+    /**
+     * A function to be called if a button is pressed.
+     * This prop, if defined on the parent SelectButtonGroup will overwrite this one
+     */
+    onSelect?: (value: string) => void;
+    /** if this button should be in a selected state */
+    isSelected?: boolean;
+    /** the color context to be applied in the selected state */
+    selectedContext?: 'neutral' | 'brand';
```

Comment:
I think better pass the entire `Context`.  Maybe in the future, we will decide to use other color schemas

Reply:
I think we can extend it later if we have design for it. The ticket currently describes only these 2.

Reply:
Now `SelectButton` component tied for some particular ticket, it should be more flexible and reusable. 
And OneUI it is open source and anyone can use it  :), thus not cleat why others Context values are not supported.

Reply:
If the effort is the same it's not a problem to support more contexts, provided that it does at least include the ones required by the ticket and those match the designs. If it's unclear how other contexts are to be styled due to lack of designs, then let's not implement them.

Reply:
ok, I will look at it specifically when it comes to css. If it is the same for all contexts, I will add them, and if designs are not clear, I will leave only the ones defined